src/wrover_status_light/src/DemoClient.py

```diff
@@ -1,19 +1,51 @@
 import rospy
 from wrover_status_light.srv import DoubleService
-from wrover_status_light.srv import DoubleServiceResponse
 from wrover_status_light.srv import DoubleServiceRequest
+from wrover_status_light.srv import DoubleServiceResponse
 
 def main():
+    '''
+    Creates a new node called "service_client"
+    '''
     rospy.init_node("service_client")
+
+    '''
+    Creates a new callable object for a service named "double_service" that is defined by the 
+    "DoubleService" class.
+    '''
     double_service = rospy.ServiceProxy("double_service", DoubleService)
+
+    '''
+    Initializes various local variables. rate specifies the frequency we would want to run at, 
+    and num is the value that is going to be doubled. 
+    '''
     rate = rospy.Rate(10)
     num = 1
 
+    '''
+    This is blocking call that waits until a service called "double_service" is created and ready.
+    '''
     rospy.wait_for_service("double_service")
+
+    '''
+    The funciton rospy.is_shutdown() checks if the module is still running or not. 
+    '''
     while not rospy.is_shutdown():
         print(num)
+
+        '''
+        Calls the service using the previously created service object and saves the response in 
+        a local variable. Similar to the server's handler function, here we simplify the code by 
+        not explicitly creating a new DoubleServiceRequest instance since it only contains one
+        field.
+        '''
         response:DoubleServiceResponse = double_service(num)
```

Comment:
Use `try` and `except` to catch if there is a problem talking to the service.

Reply:
If this demo code isn't expected to throw an exception, I would probably leave it out just to keep it simple. Exception handling may not be a concept that beginners are familiar with.

Reply:
I think it's still a good thing to include so students are aware of the need to handle exceptions when calling services. Also this service does throw an exception (integer overflow). Addressed in commit [6ae5f8d](https://github.com/WisconsinRobotics/WRoverPlayground/pull/9/commits/6ae5f8dc7bfe70413d55ac7fdfc08fa7cc6d7ff4).